pantheon-slingstart/src/main/provisioning/pantheon.txt

```diff
@@ -0,0 +1,38 @@
+[feature name=pantheon]
```

Comment:
If you look at the `pom.xml` in the `pantheon-slingstart` module, you'll see that there is a `slingstart-maven-plugin` being used. This plugin looks under the `src/main/provisioning` folder for any `.txt` files and merges them all together into a single one and finally uses that to put together the full sling distribution.

Reply:
We discussed during our 1:1 to add the appropriate directive to the pom.xml, even though we're unnecessarily specifying the default value, it will help a future developer to work out how the pieces fit together.